### DISCUSSION TITLE
Annotated Tag

### DISCUSSION THREAD

**Discussion by agusdmb** [jesseduffield/lazygit#1510] [Q&A] [answered]
Hey, first of all, I am loving this tool! it is great.

So, i use it for almost everything on my git workflow and this tool has improved my workflow speed a ton.

As the title says, i would like to know if there is any way to create annotated tags (`git tag <tag-name> -a`) with this tool, i haven't found any, so for this part of my workflow i have to go back to the console.
If there is no way to do it, then that would be a nice feature request :smiley: ...

**Comment by jesseduffield** [comment, marked as answer]
There is a PR in motion for this now: https://github.com/jesseduffield/lazygit/pull/1469

**Reply by agusdmb** [reply]
My bad, i didn't see that before. Im glad it is already a work in progress PR for this :D
